### PR TITLE
Adds gotcha for DynamoDB relating to empty strings

### PR DESCRIPTION
**Pull request by chrislennon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -983,6 +983,7 @@ DynamoDB
...
 -	It is important to make sure that DynamoDB [resource limits](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#limits-data-types) are compatible with your dataset and workload. For example, the maximum size value that can be added to a DynamoDB table is 400 KB (larger items can be stored in S3 and a URL stored in DynamoDB).
+-	DynamoDB does not allow an empty string as a valid attribute value. Main work around is to use a substitute value instead of having the field empty. This issue has been raised on the [DynamoDB Forums](https://forums.aws.amazon.com/thread.jspa?threadID=90137).
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "The most common work-around is to use a substitute value instead of leaving the field empty."

Comment (ThanosBaskous) [inline README.md]: Suggestion (for brevity): embed the link directly in the main statement. EG:

"DynamoDB does [not allow](https://forums.aws.amazon.com/thread.jspa?threadID=90137) an empty string as a valid attribute value."

**Comment by chrislennon** [comment]
Good suggestions, reads nicer! Has been changed.

**Comment by ThanosBaskous** [comment]
Thank you for your contribution, @chrislennon!

**Pull request merged**